File: PJDesignBackStageClient/src/app/features/question/feature-shared/models/get-questions.ts
import { EditResponseBase } from "src/app/shared/models/bases";
import { Category } from "src/app/shared/models/category";

export class GetQuestionsResponse extends EditResponseBase {
  id: number;
  isBefore: boolean;
  title: string;
  categories: Category[];
  createDt: Date;
  isEnabled: boolean;
}

File: PJDesignBackStageAPI/App.Model/Question/GetQuestions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class GetQuestionsResponse
    {
        public int Id { get; set; }
        public int? AfterId { get; set; }
        public bool IsBefore { get; set; }
        public string Title { get; set; } = null!;
        public DateTime CreateDt { get; set; }
        public DateTime EditDt { get; set; }
        public int EditorId { get; set; }
        public byte? EditStatus { get; set; }
        public bool IsEnabled { get; set; }
        public IEnumerable<Category>? Categories { get; set; }
    }
}
